Marlin/pins_SANGUINOLOLU_11.h

```diff
@@ -212,8 +212,12 @@
 
   #endif // !DOGLCD
 
-  #define BTN_EN1               11
-  #define BTN_EN2               10
+  #if DISABLED(ZONESTAR_LCD)
+
+    #define BTN_EN1               11
+    #define BTN_EN2               10
+
+  #endif
```

Comment:
Don't key off the universe where this panel is disabled - that will throw warnings for every other panel defines these.

Reply:
Understood.  I will rework this.